### ISSUE TITLE
Localized String with placeholder uses wrong encoding on Umlaut Argument 

### ISSUE THREAD

**Issue by maxmika** [mac-cain13/R.swift#734]
I have the following Gerrman String in my Localized.strings file:

`"rsvp_text" = "Bitte gib uns bis zum %1$s Bescheid, ob du dabei bist.";`

As you can see there's a `%1$s` argument which is recognized by R.swift perfectly fine. 
Now the string which is inserted as an argument is also localized and looks like this:

`"rsvp_date" = "01. März 2022";`
Please note the German Umlaut "ä".

The code I'm using to compose this string is the following:
`text = R.string.localizable.rsvp_text(R.string.localizable.rsvp_date())`

The problem is, the date is displayed as follows: `Bitte gib uns bis zum 01. M√§rz 2022 Bescheid, ob du dabei bist. `

So far I tried to use `.cString` versions with different encodings, different Localizeable.strings file encoding, but nothing helped. 
Basically Umlauts are working fine in "plain" strings but as soon as there's argument strings with Umlauts, they're getting messed up.
Do you know a workaround for this or can even provide a fix? 
Thank you in advance!

**Comment by tomlokhorst** [comment]
Use `%@` for strings, or in your case `%1$@`.

See:  https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/Strings/Articles/formatSpecifiers.html#//apple_ref/doc/uid/TP40004265-SW1

**Comment by maxmika** [comment]
Thank you so much, that worked. `%s` was a relict from Android Strings.
